deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java

```diff
@@ -44,6 +44,17 @@
 
 public class OpenApiClientGeneratorWrapperTest {
 
+    @Test
+    void verifySuffixPrefix() throws URISyntaxException {
```

Comment:
This is not really testing the feature. You need to check if the generated files have the right prefixes/suffixes.